src/STW.SubmissionApi.UnitTests/STW.SubmissionApi.UnitTests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+
+        <IsPackable>false</IsPackable>
+        <IsTestProject>true</IsTestProject>
+    </PropertyGroup>
+
+    <ItemGroup>
+            <PackageReference Include="FluentAssertions" Version="6.12.0"/>
+            <PackageReference Include="ILogger.Moq" Version="1.1.10"/>
+            <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0"/>
+            <PackageReference Include="Moq" Version="4.20.70"/>
+            <PackageReference Include="MSTest.TestAdapter" Version="3.3.1"/>
+            <PackageReference Include="MSTest.TestFramework" Version="3.3.1"/>
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\STW.SubmissionApi\STW.SubmissionApi.csproj"/>
```

Comment:
Is there a reason for this being here?

Reply:
Yeah cause otherwise i can't import the version controller for my tests

Reply:
Oh yeah my mistake, didn't realise it generated one for me when I made a test test project :D